### PR TITLE
pd: add an example for pd-recover

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-recover.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ cat {{/path/to}}/pd*.log | grep "idAllocator allocates a new id" |  awk -F'=' '{
 
 ### 第 4 步：使用 pd-recover
 
-只需在一个 PD 节点上执行 `pd-recover` 即可。
+只需在一个 PD 节点上执行 `pd-recover` 即可。需要注意的是，为了避免重新分配，最好在已分配 ID 增加一定值，比如从监控或者 日志获得的最大的已分配 ID 是 9000，则传入的参数最好是 10000 甚至更大。
```
Comment (hfxsd) [inline pd-recover.md:153]:
```suggestion
只需在一个 PD 节点上执行 `pd-recover` 即可。需要注意的是，为了避免重新分配，建议使用大于已分配 ID 的值，比如从监控或者日志获得的最大的已分配 ID 是 9000，则建议传入的参数是 10000 或更大。
```

**Review by HuSharp — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: pd-recover.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ cat {{/path/to}}/pd*.log | grep "idAllocator allocates a new id" |  awk -F'=' '{
 
 ### 第 4 步：使用 pd-recover
 
-只需在一个 PD 节点上执行 `pd-recover` 即可。
+只需在一个 PD 节点上执行 `pd-recover` 即可。需要注意的是，为了避免重新分配，建议使用大于已分配 ID 的值，比如从监控或者日志获得的最大已分配 ID 是 9000，则建议传入参数 10000 或更大值。
```
Comment (lilin90) [inline pd-recover.md:153]:
```suggestion
只需在一个 PD 节点上执行 `pd-recover` 即可。需要注意的是，为了避免重新分配，建议将 `-alloc-id` 设置为大于已分配 ID 的值。例如，从监控或者日志获得的最大已分配 ID 是 9000，则建议传入参数 10000 或更大值。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**